server/config/production-locales.json

```diff
@@ -13,27 +14,25 @@
       "es-AR",
       "es-CL",
       "et",
-      "eu",
       "fa",
       "ff",
       "fr",
       "fy",
       "he",
-      "hi",
```

Comment:
Not here until this is fixed: https://github.com/mozilla/fxa-content-server-l10n/issues/81